concise-mid-tag.cddl

```diff
@@ -179,10 +179,10 @@ raw-value-mask-type = bytes
 ; - How do vendor specific schemas allocate indices (keys)?
 ;   We need to specify a range of keys with rules 'first
 ;   come first served'
-endorsed-value-map = {
+endorsed-claim-map = {
```

Comment:
Perhaps the same naming can be applied on line 155 just to adhere to consistency, now that there is no endorsed-value as such in the entire file?

Reply:
nice catch.  I think we could just drop that whole comment as it's become stale.

Reply:
FYI, the assembly machinery just trashes all the comments when creating the final view.